account_invoice_customer_no_autofollow/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Invoice partner no autofollow",
```

Comment:
name should follow CamelCase convention
eg "Invoice Customer No Autofollow"

Reply:
Okey, I'll fix it